src/Apps/Search/SearchApp.tsx

```diff
@@ -0,0 +1,80 @@
+import { Col, Row, Separator, Spacer } from "@artsy/palette"
+import { SearchApp_viewer } from "__generated__/SearchApp_viewer.graphql"
+import { AppContainer } from "Apps/Components/AppContainer"
+import { HorizontalPadding } from "Apps/Components/HorizontalPadding"
+import React from "react"
+import { LazyLoadComponent } from "react-lazy-load-image-component"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import {
+  Footer,
+  RecentlyViewedQueryRenderer as RecentlyViewed,
+} from "Components/v2"
+
+export interface ArtistAppProps {
+  viewer: SearchApp_viewer
+}
+
+export class SearchApp extends React.Component<ArtistAppProps> {
+  render() {
+    const { viewer } = this.props
+
+    return (
+      <AppContainer>
+        <HorizontalPadding>
+          <Row>
+            <Col>Search Header</Col>
```

Comment:
Placeholder, but we probably dont need this (the current designs just have the count displayed and then right into the tabs and individual pages).